docs/uenv-ddt.md

```diff
@@ -0,0 +1,124 @@
+# Linaro Forge (DDT) debugger
+
+https://www.linaroforge.com/downloadForge
+
+Linaro Forge (formerly known as DDT) allows source-level debugging of Fortran, C, C++ and Python codes. It can be used for debugging serial, multi-threaded (OpenMP), multi-process (MPI) and accelerated (Cuda, OpenACC) programs running on research and production systems, including CSCS Alps system. It can be executed either as a graphical user interface or from the command-line.
+
+## Using the debugger
+
+In order to debug your code on Alps, you need to:
+
+- setup the user environment on Alps,
+- install the DDT client on your laptop,
+- build an executable with debug flags on Alps,
+- launch a job with the debugger on Alps.
+
+### Setup the environment
+
+`gcc113-mpich8125-cuda118-forge2301.squashfs` provides the latest version of Linaro Forge (23.0.1), along with gcc/11.3, cray-mpich-8.1.25 and cuda/11.8.
```

Comment:
If we are going to have one image for `forge23.01` how about calling it `ddt-23.01`?
Additionally, do we need to provide `cuda` and `cray-mpich` in the image?